### ISSUE TITLE
Feature Request: Stats Scraper 

### ISSUE THREAD

**Issue by CryptoBeano** [markrickert/cryptohopper-dashboard-watchlist#5]
Hi, 

Really like the project, removing Hoppie is a stroke of genius. I hope you are open to feature development?

I was wondering if it was possible to build a stat page scraper? I think there is information in the 'stats' page which could be presented in a better way in a tool like Excel. This would allow people to easily extract this information in efficient manner especially for people who have multiple hoppers to create a consolidated view of their portfolio. 

I'd imagine a comma separated list which could be pasted into a data tool such as Excel.

E.g : Date, Hopper Name , Number of Buys , Number of Sells, Daily Profits,  Balance at "00" hour.

**Comment by markrickert** [comment]
Hey @CryptoBeano thanks for checking out my project! I'm not ready to do anything with the stats page, because i think their stats are generally B.S. but you can see all the raw data the generates that page by logging in and then navigating to this page: https://www.cryptohopper.com/siteapi.php?todo=getstatspage

**Comment by Leebo-missile** [comment]
I agree and wish something could be done to their stats page, anything is better than what currently is provided, trade history and stats in general.
